styles/pbwow3/template/mchat_body.html

```diff
@@ -0,0 +1,127 @@
+{% if MCHAT_PAGE in ['custom', 'archive'] %}
+	{% INCLUDE 'overall_header.html' %}
+{% endif %}
+
+{% EVENT dmzx_mchat_overall_header_after %}
+
+{% EVENT dmzx_mchat_header_before %}
+
+{% if MCHAT_PAGE == 'custom' %}
+   	<h2>{{ lang('MCHAT_CUSTOM_PAGE') }}</h2>
+{% elseif MCHAT_PAGE == 'archive' %}
+	<h2>{{ lang('MCHAT_ARCHIVE_PAGE') }}</h2>
+{% endif %}
+
+{% EVENT dmzx_mchat_header_after %}
+
+{% INCLUDE '@dmzx_mchat/mchat_script_data.html' %}
+
+{% if MCHAT_PAGE == 'archive' and (loops.pagination|length or MCHAT_TOTAL_MESSAGES) %}
+<div class="action-bar bar-top">
+	{% EVENT dmzx_mchat_action_bar_top_before %}
+	<div class="pagination">
+		{{ MCHAT_TOTAL_MESSAGES }}
+		{% if loops.pagination|length %}
+			{% INCLUDE 'pagination.html' %}
+		{% else %}
+			&bull; {{ PAGE_NUMBER }}
+		{% endif %}
+	</div>
+	{% EVENT dmzx_mchat_action_bar_top_after %}
+</div>
+{% endif %}
+
+{% EVENT dmzx_mchat_before %}
+
+{% if MCHAT_IS_COLLAPSIBLE %}
+	<a class="category{% if S_MCHAT_HIDDEN %} hidden-category{% endif %} mchat-category"></a>
+{% endif %}
+
+<div class="mchat-wrapper collapse-box">
+	{% INCLUDE '@dmzx_mchat/mchat_header.html' %}
```

Comment:
`mchat_body.html` shouldn't be copied from prosilver's folder as it would probably be forgotten when updating it in the future.

`MCHAT_IS_COLLAPSIBLE` comes from the Collapsible Categories extension which isn't needed in PBWoW3, so it will always evaluate to `false` anyway. For the remaining elements, is there a way to fix the styling with CSS only?

Also you removed `<a id="mChat"></a>` but it's an essential element.